applications/luci-app-filemanager/htdocs/luci-static/resources/view/system/filemanager.js

```diff
@@ -0,0 +1,2313 @@
+			if (statusInfo) statusInfo.textContent = _('No item selected.');
+			if (statusProgress) statusProgress.innerHTML = '';
+		}).catch(function(err) {
+			ui.addNotification(null, E('p', _('Failed to save changes to %s "%s": %s').format(_('item'), newItemName, err.message)), 'error');
+		});
+	},
+
+	// Handler for saving interface settings
+	handleSaveSettings: function(ev) {
+		// Save new settings and update the interface
+		ev.preventDefault();
+		var self = this;
+		var inputs = {
+			columnWidths: document.getElementById('column-widths-input'),
+			columnMinWidths: document.getElementById('column-min-widths-input'),
+			columnMaxWidths: document.getElementById('column-max-widths-input'),
+			padding: document.getElementById('padding-input'),
+			paddingMin: document.getElementById('padding-min-input'),
+			paddingMax: document.getElementById('padding-max-input'),
+			currentDirectory: document.getElementById('current-directory-input'),
+			windowWidth: document.getElementById('window-width-input'),
+			windowHeight: document.getElementById('window-height-input')
+		};
+
+		function parseWidthSettings(inputValue, configKey) {
+			if (!inputValue) return;
+			inputValue.split(',').forEach(function(widthStr) {
+				var widthParts = widthStr.split(':');
+				if (widthParts.length === 2) {
+					var field = widthParts[0];
+					var width = parseInt(widthParts[1], 10);
+					if (!isNaN(width)) {
+						config[configKey][field] = width;
+					}
+				}
+			});
+		}
+		if (inputs.columnWidths && inputs.padding) {
+			parseWidthSettings(inputs.columnWidths.value.trim(), 'columnWidths');
+			parseWidthSettings(inputs.columnMinWidths.value.trim(), 'columnMinWidths');
+			parseWidthSettings(inputs.columnMaxWidths.value.trim(), 'columnMaxWidths');
+			var paddingValue = parseInt(inputs.padding.value.trim(), 10);
+			var paddingMinValue = parseInt(inputs.paddingMin.value.trim(), 10);
+			var paddingMaxValue = parseInt(inputs.paddingMax.value.trim(), 10);
+			if (!isNaN(paddingValue)) {
+				config.padding = paddingValue;
+			}
+			if (!isNaN(paddingMinValue)) {
+				config.paddingMin = paddingMinValue;
+			}
+			if (!isNaN(paddingMaxValue)) {
+				config.paddingMax = paddingMaxValue;
+			}
+			if (inputs.currentDirectory) {
+				var currentDirectoryValue = inputs.currentDirectory.value.trim();
+				if (currentDirectoryValue) {
+					config.currentDirectory = currentDirectoryValue;
+				}
+			}
+			if (inputs.windowWidth && inputs.windowHeight) {
+				var windowWidthValue = parseInt(inputs.windowWidth.value.trim(), 10);
+				var windowHeightValue = parseInt(inputs.windowHeight.value.trim(), 10);
+				if (!isNaN(windowWidthValue)) {
+					config.windowSizes.width = windowWidthValue;
+				}
+				if (!isNaN(windowHeightValue)) {
+					config.windowSizes.height = windowHeightValue;
+				}
+			}
+			saveConfig().then(function() {
+				ui.addNotification(null, E('p', _('Settings uploaded successfully.')), 'info');
```

Comment:
Settings saved

Reply:
rebase on master. I've extended addNotification with a timeout parameter, so that notifications remove themselves automatically. Rework your 'info' type notifications to include this.

Reply:
I don't see the timeout number in use: this means that all of those info popups about settings saved and file uploaded stick around pushing the GUI further down:

For a five second timeout try:
```js
				ui.addNotification(null, E('p', _('Settings saved.')), 5000, 'info');
```